tree/treeplayer/test/dataframe/dataframe_cache.cxx

```diff
@@ -31,6 +31,8 @@ TEST(Cache, FundType)
    }
 }
 
+/*
+// The contiguity has been dropped since now caching is backed up by a TDS
```

Comment:
i'd add something like `// FIXME readd test when TLazyDS does not make internal copies anymore`